### PR TITLE
add mlflow to train script for these tasks: token-cls, audio-cls, sem…

### PR DESCRIPTION
**Pull request by HDThang**
# What does this PR do?
add mlflow and log num_params to train script for these tasks:
- token classification
- audio classification
- semantic segmentation
- image pretraining

### CODE REVIEW

File: examples/pytorch/audio-classification/run_audio_classification.py [hunk 1]
```diff
@@ -391,6 +408,9 @@ def compute_metrics(eval_pred):
             checkpoint = last_checkpoint
         train_result = trainer.train(resume_from_checkpoint=checkpoint)
         trainer.save_model()
+        metrics['throughput'] = metrics['train_samples_per_second']
+        metrics['loss']= metrics['train_loss']
```
Comment (dohyung-kim-moreh) [inline examples/pytorch/audio-classification/run_audio_classification.py:412]: As you know, throughput, loss, lr must be logged in every log-interval.
It seems logging once after training is done..?

**Comment by dohyung-kim-moreh** [comment]
Please add to log Evaluation metrics according to https://www.notion.so/Benchmark-Code-Guideline-1c544796906a4ccea337ae289dab5b68?pvs=4

File: examples/pytorch/audio-classification/run_audio_classification.py (hunk 1 above)
Comment (HDThang) [reply examples/pytorch/audio-classification/run_audio_classification.py:412]: That's right, the current code save the metric at the end of the training.

**Comment by HDThang** [comment]
The eval metrics are already logged in Metrics by mlflow.

File: examples/pytorch/audio-classification/run_audio_classification.py (hunk 1 above)
Comment (HDThang) [reply examples/pytorch/audio-classification/run_audio_classification.py:412]: Using the trainer can only log loss and lr each steps, throughput/train_samples_per_second is logged at the end of the training process. If those metrics are really need to be logged at each steps, please let me know and I will work on it.

File: examples/pytorch/audio-classification/run_audio_classification.py [hunk 2]
```diff
@@ -382,6 +393,12 @@ def compute_metrics(eval_pred):
         tokenizer=feature_extractor,
     )
 
+    # Mlflow initial
+    #set the os enviroment for MLflowCallback
+    os.environ["DISABLE_MLFLOW_INTEGRATION"] = "False"
```
Comment (yeomni) [inline examples/pytorch/audio-classification/run_audio_classification.py:398]: Please consider using a formatter such as yapf or black. I'd recommend black.

File: examples/pytorch/audio-classification/run_audio_classification.py (hunk 1 above)
Comment (HDThang) [reply examples/pytorch/audio-classification/run_audio_classification.py:412]: On research, I found that the trainer log loss, and lr each step at .state.log_history() not .metric(). However, the throughput is not logged though, I'll find a way to add log it each log interval
